<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Demo</class>
 <widget class="QMainWindow" name="Demo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ABCD COMMAND POINT</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../_images/aperture-1142967_960_720.png</normaloff>../_images/aperture-1142967_960_720.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="IST_label">
       <property name="text">
        <string>Image Sequence Title:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="IST_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ICI_label">
       <property name="text">
        <string>Image Capture Interval:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ICI_spinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ISD_label">
       <property name="text">
        <string>Image Sequence Duration:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="ISD_spinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="Lower_Bay_Display">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>350</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_0_Label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_2_Label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_6_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_4_Label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_1_Label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_3_Label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_5_Label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_7_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="Upper_Bay_Display">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>350</width>
      <height>161</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>331</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_8_Label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_10_Label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_14_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_12_Label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_9_Label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_11_Label">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_13_Label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="Unit_15_Label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../_images/stop-red.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Upper_Bay_Label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>190</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Upper Bay Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="Lower_Bay_Label">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Lower Bay Status</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>293</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="email_label">
       <property name="text">
        <string>Email Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="email_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_email">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Confirm</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>711</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="Update_Status">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Updating Status...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Start_Imaging">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Start Imaging</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
